cmd/maestro/common/otlp_sdk.go

```diff
@@ -0,0 +1,98 @@
+package common
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"go.opentelemetry.io/contrib/exporters/autoexport"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/propagation"
+	"go.opentelemetry.io/otel/sdk/resource"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.25.0"
+
+	errors "github.com/zgalor/weberr"
+	"go.uber.org/zap"
+
+	"github.com/openshift-online/maestro/pkg/constants"
+)
+
+// Without a specific configuration, a noop tracer is used by default.
+// At least two environment variables must be configured to enable trace export:
+//   - name: OTEL_EXPORTER_OTLP_ENDPOINT
+//     value: http(s)://<service>.<namespace>:4318
+//   - name: OTEL_TRACES_EXPORTER
+//     value: otlp
```

Comment:
nice, we can make it work easily in our deployment for ARO HCP by passing in the MDSD endpoint of the cluster